src/main/java/ag/boersego/bgjs/BGJSGLView.kt

```diff
@@ -106,16 +114,18 @@ open class BGJSGLView(engine: V8Engine, val textureView: V8TextureView) : JNIV8O
             callbacksForThisRedraw.addAll(callbacksRedraw)
             queuedAnimationRequests.forEach { callbacksForThisRedraw.add(it.cb) }
             queuedAnimationRequests.clear()
-        }
 
-        didDraw = !callbacksForThisRedraw.isEmpty()
+            didDraw = !callbacksForThisRedraw.isEmpty()
+
+            // TODO: what are on.redraw?
```

Comment:
See explanation provided via messenger; should definitely not be called here.
Instead we should have something like "fun requestRedraw" that would trigger all the callbacks.
This method would be unused for now, but could be called by the app, or by the library in the future should the texture view be reset for any reason other than resizing..